docs/dxp/7.x/en/site-building/creating-pages/master-pages.md

```diff
@@ -1,27 +1,52 @@
 
-A Master page Template provides the common elements for a page (Headers, Footers, etc.), while the Theme provides the overall look and feel for the Site's pages. Prior to Liferay DXP 7.3, these common elements were provided by the Theme, which means that any changes that needed to be made required an update to the Theme. Providing the common elements through a Theme poses a few limitations:
+## Master Page Templates and Themes
 
-* Only users with System Administrator permissions can apply changes, since they must deploy the Theme.
+Themes are still available in 7.3+ and they differ from MBT's in the following crucial ways:
+
+| Capability | MBT | Theme |
+| --- | --- | --- |
+| Requires Plugin Deployment to update changes |  | x |
+| Something you can only do with a theme |  | x |
+| Something you can only do with a MBT | x |  |
+
+-->
+
+A Master Page Template provides the common elements for a page (Headers, Footers, etc.), while a Theme provides the overall look and feel for the Site's pages. Prior to Liferay DXP 7.3, these common elements were provided by the Theme, which means that any changes that needed to be made required an update to the Theme. Providing the common elements through a Theme poses a few limitations:
+
+* System Administrators are required when a theme must be updated and re-deployed.
 
 * Themes are available for all virtual instances and Sites, so you can't make a Theme available to certain customers, while hiding it from others.
 
 * Common Theme element (Headers, Footers, etc.) customization options are limited to what the Theme developer provides (i.e. color schemes, theme settings, etc.).
 
 Master Page Templates solve all these limitations by letting you create common elements that are unique per Site. The remainder of the page is left editable and is built like any other [Content Page](./04-building-content-pages.md). Each Page and Page Template must have an associated Master Page Template. By default, this is the "Blank" Master, which doesn't provide any common elements.
 
+<!-- The highlighted portion of this screenshot looks...messy? -->
+
 ![The Blank Master Page Template is the default for Pages, and Page Templates.](./master-pages/images/01.png)
 
+<!-- I'd remove this note because while it is a difference, it's a relatively subtle and obvious difference from how the UI appeared in the past.
+
 ```note::
-    In Liferay DXP 7.3+, Page Templates and Display Page Templates have been moved out of "Pages" and into their own separate application called "Page Templates" under "Site Builder".
+   In Liferay DXP 7.3+, Page Templates and Display Page Templates have been moved out of "Pages" and into their own separate application called "Page Templates" under "Site Builder".
 ```
+-->
 
```

Comment:
> I'd remove this note because while it is a difference, it's a relatively subtle and obvious difference from how the UI appeared in the past.

Yeah, I suppose you're right